lms/static/js/ccx/schedule.js

```diff
@@ -53,6 +53,8 @@ var edx = edx || {};
 	        self.render();
 	    });
 
+	    self.disabled_fields($('.ccx_due_date_time_fields'), true);
+	    self.disabled_fields($('.ccx_start_date_time_fields'), true);
```

Comment:
Why are you disabling these fields here, and elsewhere? Please add comments to explain why things are being disabled/enabled throughout this file.